gitops/README.md

```diff
@@ -0,0 +1,65 @@
+# Initialize the GitOps environment
+## Install the operators
+The `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators can be installed from the solution 
+derived from the [Janus IDP Demo](https://github.com/redhat-gpte-devopsautomation/janus-idp-bootstrap)
+>This repository contains automation to install the Janus IDP Demo, as well as supporting components
+
+A fork has been created to remove the configuration that excludes Tekton resources from being configured from the 
```

Comment:
I'd suggest to highlight why we need a fork.
For example:
"To address the need for including Tekton resources within the ArgoCD applications, a fork has been initiated to remove the exclusion configuration"